tests/testfiles/lunamark-markdown/no-gfm-auto-identifiers.test

```diff
@@ -0,0 +1,86 @@
+\def\markdownOptionAutoIdentifiers{false}
```

Comment:
I am in favor of removing both `no-auto-identifiers.test` and `no-gfm-auto-identifiers.test`.